packages/control-property-editor/development/public/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
Fixtures used for testing - either automatic or manual - are put into the `test` folder, could you please move these as well into `test/fixtures`?

Reply:
I have moved. except for manual mocks
`packages/control-property-editor-ui5/test/__mocks__`